blocks/section-title-block/features/section-title/_children/section-title.jsx

```diff
@@ -16,14 +16,18 @@ const StyledLink = styled.a`
 
 const Separator = '  \u00a0 • \u00a0  ';
 
+// using the class twice to ensure the • and the text are both same font-size and
 const styledLinkTmpl = (name, id, separator, arcSite) => (
-  <StyledLink
-    primaryFont={getThemeStyle(arcSite)['primary-font-family']}
-    href={id}
-    key={id}
-  >
-    {`${name}${separator}`}
-  </StyledLink>
+  <span key={id} className="section-title--styled-link">
+    <StyledLink
+      primaryFont={getThemeStyle(arcSite)['primary-font-family']}
```

Comment:
Would be nice if this was switched over to the shared styles primary font component

Reply:
shared styles primary font for the link? oh, like pass in as an a tag

Reply:
Yeah, the shared component has the ability to render `as` any HTML element.

This would then become

```
<PrimaryFont as="a" href={id} className="section-title--styled--link">
{name}
</PrimaryFont>
```

Reply:
right, makes sense. Updated @matthewroach